templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-backup-sh.yaml

```diff
@@ -16,11 +16,18 @@ contents:
       exit 1
     fi
 
+    usage () {
+        echo 'Path to backup file needed: ./etcd-snapshot-backup.sh ./backup.db'
```

Comment:
s/needed/required?

Reply:
+1

Reply:
+1

Reply:
I would like the message to include the string "Usage:" somewhere. For example.,
```echo 'Path to backup file is needed.\n Usage: ./etcd-snapshot-backup.sh <path-to-backup-file>'```